docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
+  language of each format and then package the application in those distributions, and/or
+  ignore most distributions and package and support a couple of distributions. This makes
+  it very difficult to innovate the Linux desktop as a whole as a considerably high amount of time
+  is spent on packaging.
+- **Limited to apps that are packaged**: Not all applications are natively available
+  in every Linux distribution. If an application is not available in a specific
+  distribution, the user will have to rely on manually downloading the archive
+  of the application, extracting it and hoping the application will launch.
+- **Limited to distributions that have the apps**: The user is limited to the
+  number of distributions that have the needed applications for them
+  to properly setup their workflow. This reduces the amount of distributions
+  that can be suitable for a user.
+- **Hard to innovate in OS space**: The maintainers of the distributions have to spend a lot of
+  time packaging applications to make the distribution suitable for the end user, instead of focusing
+  on their end goals. This delays the progress of each distribution.
+
+Flatpak strives to fix the issues listed above, by conveniently enabling developers to distribute
+applications from one source and to target the entire Linux desktop.
+
 Reasons to use Flatpak
 ----------------------
 
-Flatpak has some major advantages over other approaches to distributing
-applications on Linux. First and foremost, Flatpak allows a single application
-build to be installed and run on virtually any Linux distribution. It can also
-be used in combination with `Flathub <https://flathub.org/>`_, a centralized
-service for distributing applications on all distributions. This makes it
-possible for application developers to target the entire Linux desktop market
-from one place.
+Flatpak has some major advantages over most system package managers:
+
+- **Universality**: Flatpak allows applications to be installed and run on virtually any Linux
+  distribution. This includes non-GNU distributions, systemd-free distributions,
+  distributions with a read-only operating system (OS), and systems using a different architecture.
+- **Space for innovations**: Flatpak facilitates distribution maintainers to focus on their goals
+  to innovate their distribution.
+- **Stability**: Breakage in a Flatpak application will not risk the system from breaking.
+  This is because Flatpak applications and runtimes are contained to not interfere
+  with the system altogether.
+- **Rootless install**: Elevated privileges are not required when installing a Flatpak
+  application or a runtime.
 
-Flatpak also offers native integration for the main Linux desktops, so that
-users can easily browse, install, run and use Flatpak applications through
-their existing desktop environment and tools.
+Flatpak has some major advantages over other universal approaches to distributing
```

Comment:
This wording implies that other universal package managers don't have a centralized store, which isn't true.

Reply:
That's why I wrote "universal approaches to distributing" and not "universal package managers". AppImage for example doesn't really have a centralized store, unless I'm wrong.

Reply:
Oh wow, nevermind. https://appimage.github.io/apps/